data/window-print-android.yml

```diff
@@ -0,0 +1,46 @@
+title: Missing window.print support in GeckoView
+severity: normal
+user_base_impact: unknown
+tags:
```

Comment:
Wonder if we can estimate this to be `large` given the amount of reports and the fact that important sites are affected (delta, amazon, allrecipes, amex travel, medicare, etc.)

Reply:
I thought about that, but I haven't looked into "hard" data yet. But with you agreeing, let's set it to large.